site/content/en/docs/tasks/manage/setup_failure_recovery.md

```diff
@@ -0,0 +1,211 @@
+---
+title: "Setup failure recovery"
+date: 2025-12-04
+weight: 12
+description: >
+  Configure failure recovery mechanisms to unblock workloads after a node outage.
+---
+
+This describes and shows how to configure the failure recovery mechanism that automatically transitions pods
+that are stuck terminating into the `Failed` phase. This is especially relevant for `Job`-based workloads that
```

Comment:
Just to double check: "pods stuck terminating" does _imply_ a node failure, right? Or not necessarily?
(My concern is to not over-promise about this feature in the 1st paragraph).